dlt/common/destination/utils.py

```diff
@@ -226,7 +227,9 @@ def prepare_load_table(
         if table_format == "native":
             # no special table format, use whatever is supported
             prep_table["table_format"] = None
-
+        # remove incomplete columns
+        for column, _ in find_incomplete_columns(table):
+            prep_table["columns"].pop(column["name"], None)
```

Comment:
why do you need `pop(..., None)` here? column with `name` 100% exists. were you getting errors?